package.json

```diff
@@ -31,7 +31,7 @@
     "lint:package-jsons": "tsx ./scripts/lint-package-jsons",
     "prettier:check": "prettier --check packages",
     "prettier:format": "prettier --write packages",
-    "changeset:publish": "changeset publish --no-git-tag",
+    "changeset:publish": "tsx ./scripts/changeset/changeset-publish",
```

Comment:
This one is only using synchronous operations and can be a regular `.ts` and the extension doesn't have to be specified.

In the case of `changeset:update-changelog`, the file is `.mts` because it's got top-level asynchronous code which isn't allowed in `.ts` files. I wrote it like that because I wanted to await the execution of the asynchronous code and not just fire and forget the "main" function. But perhaps I misunderstood how `Promise`s are handled in the CLI. If the process awaits their execution before it exits then that file could be converted to `.ts` as well.